setup.cfg

```diff
@@ -43,7 +43,7 @@ install_requires =
   Pillow>=6.2.2,!=9.2.*  # minimum version tested there: https://github.com/PyFPDF/fpdf2/actions/runs/2295868575
   # Version 9.2.0 is excluded due to DoS vulnerability with TIFF images: https://github.com/PyFPDF/fpdf2/issues/628
   # Version exclusion explained here: https://devpress.csdn.net/python/630462c0c67703293080c302.html
-  fonttools>=4.34.0
```

Comment:
I agree that we should support fonttools>=4.41.1,
but why should we set this as the minimal version?

`fpdf2` can be used with older versions of `fonttools`,
so this change will restrain `fpdf2` compatibility with `fonttools`,
which could be annoying for end-users that need to use `fpdf2` with older versions of `fonttools`.

I'd prefer to only define there in `setup.cfg` the very absolute minimal versions of our dependencies that `fpdf2` can work with.

Also, do you know what change in a `fonttools` release generated those differences in reference PDF files?
I saw this in [the release notes for v4.39.0](https://github.com/fonttools/fonttools/releases/tag/4.39.0):

> [feaLib] Fixed handling of ignore statements with unmarked glyphs to match makeotf behavior, which assumes the first glyph is marked

Reply:
Some thoughts here: https://github.com/PyFPDF/fpdf2/pull/863#issuecomment-1648688622